tutorials/stats-sensor-space/plot_stats_cluster_1samp_test_time_frequency.py

```diff
@@ -106,19 +130,24 @@
     if p_val <= 0.05:
         T_obs_plot[c] = T_obs[c]
 
+# Just plot one channel's data
+ch_idx, f_idx, t_idx = np.unravel_index(
+    np.nanargmax(np.abs(T_obs_plot)), epochs_power.shape[1:])
+# ch_idx = tfr_epochs.ch_names.index('MEG 1332')  # to show a specific one
```

Comment:
cruft? or intentional?

Reply:
Intentional in case people want to play with it in the example (it used to only show that sensor)